assets/js/edit.js

```diff
@@ -275,25 +377,105 @@ class Edit extends Component {
 						</PanelRow>
 					</PanelBody>
 				</InspectorControls>
-				<div className={className}>
+				<div className="wp-block-podcasting-podcast-outer">
 					{src ? (
-						<figure key="audio" className={className}>
-							<audio controls="controls" src={src} />
-							{((caption && caption.length) || !!isSelected) && (
-								<RichText
-									tagName="figcaption"
-									placeholder={__(
-										'Write caption…',
-										'simple-podcasting'
+						<>
+							<div className="wp-block-podcasting-podcast__container">
+								{showImage && displayArt && (
+									<div className="wp-block-podcasting-podcast__show-art">
+										<div className="wp-block-podcasting-podcast__image">
+											<img
+												src={showImage}
+												alt={showName}
+											/>
+										</div>
+									</div>
+								)}
+
+								<div className="wp-block-podcasting-podcast__details">
+
+									{displayEpisodeTitle && (
+										<h3 className="wp-block-podcasting-podcast__show-title">
+											{displayEpisodeNumber && (
```

Comment:
```suggestion
											{displayEpisodeNumber && episodeNumber && (
```

Hide if episode number isn't filled out.